### PR TITLE
Enhanced README.md

### PR DESCRIPTION
**Pull request by nvinayvarma189**
added badges,Screen shots with few emojis to make it look better and it is more understandable for absolute beginners.

### CODE REVIEW

**Comment by nvinayvarma189** [comment]
Please review this PR and kindly notify me if changes are requested. :grinning: @dibyadas

File: README.md [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+To install this chrome extension. <br>
+1.Click on the "clone or download" button which is on the top-right of the page.<br>
+![alt text](https://github.com/nvinayvarma189/read-it/blob/master/img/pic1.png)<br>
+2.After you downloaded it , Extract the files into your system .<br>
+3.launch chrome and visit `chrome://extension`<br>
+4.Make sure you enable the Developer mode, only then you can load the packages.<br>
+![alt text](https://github.com/nvinayvarma189/read-it/blob/master/img/pic2.png)
+5.Click on load unpacked extension and choose the folder that you had extracted.<br>
+6.Then you can see this<br>
+7.Click on "Allow in incognito" and it is safe. :100: <br>
```
Comment (dibyadas) [inline README.md]: Can you just remove the 💯  here ?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
+![alt text](https://github.com/nvinayvarma189/read-it/blob/master/img/pic3.png)<br>
 
-To install the Chrome extension, fire up Google Chrome (`google-chrome`)  
-Navigate to `Settings` -> `Extensions` and check `Developer mode`.  
-Choose `Load unpacked extension...` and browse to the cloned folder.
+## Contributing
+If you find an issue regarding this project or if you feel that you can enchance something, then you are free to report an issue in the ![issues page](https://github.com/dibyadas/read-it/issues "issues").Please check if what you feel to be improved is already being implemented or not. Then you can **fork** the repo and you can edit it.After you are done with it, you can now submit a PR(pull request). :grinning: :+1:<br>
```
Comment (dibyadas) [inline README.md]: Change the 😀 to a simple smile.

**Review by dibyadas — APPROVED** [review]
This looks good! Thanks.

**Comment by nvinayvarma189** [comment]
I made the requested changes. Can you please review it now @dibyadas

**Review by dibyadas — APPROVED** [review]
Nice. Thanks

**Pull request merged**